src/randoop/operation/ArgumentParser.java

```diff
@@ -0,0 +1,36 @@
+package randoop.operation;
+
+import randoop.types.TypeNames;
+
+/**
+ * ArgumentParser recognizes method/constructor arguments in a string
```

Comment:
What kind of string?  A method signature?  In what format?  Also, do you mean it recognizes parameters or arguments?  Or maybe parameter types?  Please indicate the exact format expected of the input, and give a concrete example string in the documentation.
Don't forget to have some kind of documentation for the method itself, as well.